cli/copr_cli/main.py

```diff
@@ -853,6 +857,7 @@ def action_add_or_edit_package_custom(self, args):
             "script_builddeps": args.script_builddeps,
             "script_resultdir": args.script_resultdir,
             "max_builds": args.max_builds,
+            "timeout": args.timeout,
             "webhook_rebuild": ON_OFF_MAP[args.webhook_rebuild],
```

Comment:
You will have to add the parameter to `parser_add_or_edit_package_parent`. Otherwise

```
$ copr-cli edit-package-scm ... --timeout 60
copr: error: unrecognized arguments: --timeout 60
```